src/components/Icon/svgs/LoungeAccessIcon.tsx

```diff
@@ -1,26 +1,25 @@
-import PropTypes from 'prop-types';
 import * as React from 'react';
 import Svg, {G, Path, Polygon} from 'react-native-svg';
 import themeColors from '@styles/themes/default';
 
-const propTypes = {
+type LoungeAccessIconProps = {
     /** The fill color for the icon. Can be hex, rgb, rgba, or valid react-native named color such as 'red' or 'blue'. */
-    fill: PropTypes.string,
+    fill?: string;
 
     /** Is icon hovered */
-    hovered: PropTypes.string,
+    hovered?: string;
 
     /** Is icon pressed */
-    pressed: PropTypes.string,
+    pressed?: string;
```

Comment:
I know that it was like that before , but this seems super od for me , why its string? 😄